backend/blueprint/api/utils/mail.py

```diff
@@ -0,0 +1,34 @@
+from flask_mail import Mail, Message
+from flask import current_app as app
+from flask import jsonify
+from models.user import User
+from .response import Response
+
+
+def send_mail(message):
+    if message and message.receipent:
+        user = User.getUser(message.receipent)
+        if not user:
+            return jsonify(
+                Response(403).generateMessage(
+                    'Couldn\'t find the username specified'))
+        try:
+            msg = Message(
+                subject=message.subject,
+                sender=app.config['MAIL_USERNAME'],
+                recipients=[message.receipent],
+                body=message.body)
+            Mail(app).send(msg)
+        except Exception as e:
+            print(e)
```

Comment:
You are already using it in sending Response, so I guess we can remove thie `print` statement?
What say?